sdk/lusid/api/transaction_portfolios_api.py

```diff
@@ -169,10 +203,11 @@ def adjust_holdings_with_http_info(self, scope, code, effective_at, adjust_holdi
 
         # Authentication setting
         auth_settings = ['oauth2']  # noqa: E501
-
-        # set the LUSID header
-        header_params['X-LUSID-SDK-Language'] = 'Python'
-        header_params['X-LUSID-SDK-Version'] = '0.11.3042'
```

Comment:
What happened to these header params? We need these in here for logging.

Reply:
Should be back again

Reply:
Do we need this back given what you said about it being in ApiClient now?

Reply:
Once we go to the new template, these will be removed again as they are now in the api_client